ui/src/app/components/Timer/timer.tsx

```diff
@@ -0,0 +1,27 @@
+import { ActionButton } from "argo-ui/v2";
+import * as React from 'react';
+
+export const Timer = (props : { date : string}) => {
+    const [time, setTime] = React.useState(0);
+    React.useEffect(() => {
+        let timer: any;
+        let start = Math.floor(((new Date(props.date).getTime()-Date.now())/1000));
+        setTime(start);
+        timer = setInterval(() => {
+           start--;
+           if(start === 0 ) {
+               clearInterval(timer);
+               start = 0;
+           }
+           setTime(start);
+       }, 1000);   
+       return () => {
+         if(timer) {
+            clearInterval(timer);
+         }
+       }
+   }, [props.date]);
+   return (
+      time === 0  ? null: <ActionButton label={`delay: ${time}s`} />
```

Comment:
Looks like this component is not just Timer. It has a button and `delay` label so it should be renamed accordingly. 

The `argo-ui` has `Ticker` component. So might consider just using it and "inline" implementation into `pods.tsx`. E.g.: 


pods.tsx
```tsx
import {Ticker} from 'argo-ui';

...
  <Ticker>
  {now => {
    // calculate time here
    return time === 0 ? null: <ActionButton label={`delay: ${time}s`} />;
  }}
...
```

Reply:
Thanks Alex, I will check and update.